dd-sdk-android-fresco/README.md

```diff
@@ -0,0 +1,72 @@
+# Datadog Integration for Fresco
+
+## Getting Started 
+
+To include the Datadog integration for [Fresco][1] in your project, simply add the
+following to your application's `build.gradle` file.
+
+```
+repositories {
+    maven { url "https://dl.bintray.com/datadog/datadog-maven" }
+}
+
+dependencies {
+    implementation "com.datadoghq:dd-sdk-android:<latest-version>"
+    implementation "com.datadoghq:dd-sdk-android-fresco:<latest-version>"
+}
+```
+
+### Initial Setup
+
+Before you can use the SDK, you need to setup the library with your application
+context and your API token. You can create a token from the Integrations > API
+in Datadog. **Make sure you create a key of type `Client Token`.**
+
+```kotlin
+class SampleApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        val config = DatadogConfig.Builder(BuildConfig.DD_CLIENT_TOKEN).build()
```

Comment:
I made the same mistake for Glide, we need to add the `envName` and `applicationId` in our sample code in our Readme. If they copy paste it as is, it won't compile.

Reply:
I will fix it also in Glide